### PR TITLE
feat: add bold, italic and h2 header

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Add bold, italic and h2 header to email templates.

Looked at how Veteran Affairs did it in the US
- https://github.com/department-of-veterans-affairs/notification-utils/commit/fc4e518ab2e86dbd36dc5cc8f11b638746d83e11
- https://github.com/department-of-veterans-affairs/notification-utils/commit/38700405b2c56e2d680b54b30dc3e4f6d091da17

Screen Shot 2021-05-04 at 10 57 56

TODO: pull it this new version on admin and API. ⚠️ does not have a CSS rule for `<strong>` yet

### CODE REVIEW

File: notifications_utils/logging.py
Comment (AntoineAugusti) [inline notifications_utils/logging.py:60]: I'm sorry, changes here are not related to templates. The logic here was broken when `time_taken` was not defined, the call to the method `record_stats` was failing because the variable was not defined.

I took another look at the code and discovered that time_taken is expected to be in milliseconds. Values reported to StatsD were wrong.

https://github.com/jsocol/pystatsd/blob/f3f304b4b2c3d5eddeb9f4977d9c82c64c37a052/statsd/client/base.py#L26-L35

**Review by QuanNguyen-CDS — APPROVED** [review]

**Comment by AntoineAugusti** [comment]
@jimleroyer Can you approve this PR one more time? @QuanNguyen-CDS does not have write access anymore so I cannot merge this PR.

**Review by jimleroyer — APPROVED** [review]

**Pull request merged**
